pydata/apps.py

```diff
@@ -0,0 +1,85 @@
+import re
+
+from django.apps import apps, AppConfig
+from django.utils.functional import curry
+
+
+class PyDataConfig(AppConfig):
+    name = 'pydata'
+    label = 'PyData'
+    verbose_name = 'AMY for PyData conferences'
+
+    def ready(self):
+        from . import checks
+
+        Sponsorship = apps.get_model('workshops', 'Sponsorship')
+        # Add choices to the `amount` field
+        Sponsorship.LEVELS = (
+            (0, 'Founding'),
+            (15000, 'Diamond'),
+            (8000, 'Platinum'),
+            (5000, 'Gold'),
+            (3000, 'Silver'),
+            (1500, 'Supporting'),
+            (1, 'Community'),
+        )
+        amount_field = Sponsorship._meta.get_field('amount')
+        amount_field.choices = Sponsorship.LEVELS
+        amount_field.verbose_name = 'Sponsorship level'
```

Comment:
@narayanaditya95 I reset the `develop` branch, but unfortunately that didn't reopen this PR. I'm really sorry about this mess, @narayanaditya95, but can you make a new PR with these changes?